cypress.config.ts

```diff
@@ -5,7 +5,6 @@ import { setupHardhatEvents } from 'cypress-hardhat'
 export default defineConfig({
   projectId: 'yp82ef',
   videoUploadOnPasses: false,
-  waitForAnimations: false,
```

Comment:
Is this intentional? I think it was added recently by Zach to speed things up

Reply:
yeah it was to fix a flakey test but this has been added on main already so I can rebase

Reply:
Ah, sorry, I misread. I thought it was the opposite. Yeah, this is good change!